example/lib/main.dart

```diff
@@ -12,15 +12,11 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  String _platformVersion = 'Unknown';
-
   @override
   initState() {
     super.initState();
-    StripeSource.setPublishableKey("pk_test");
-    StripeSource.addSource().then((String token) {
-      _addSource(token);
-    });
+
+    StripeSource.setPublishableKey("sk_0G1JZIKlGks6b8DhZUf5vfJY7XCMq");
```

Comment:
please revert this... :)

Reply:
oh snap, thanks that definitely wasn't supposed to be there

Reply:
I'll squash while merging 👍

Reply:
It was a test key and not a public one FYI, but I still needed to remove it